[2025-07-24T17:44:55.714+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tune_stream_dag.Load_user_dim_table scheduled__2023-01-07T13:00:00+00:00 [queued]>
[2025-07-24T17:44:55.781+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tune_stream_dag.Load_user_dim_table scheduled__2023-01-07T13:00:00+00:00 [queued]>
[2025-07-24T17:44:55.786+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2025-07-24T17:44:55.849+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Load_user_dim_table> on 2023-01-07 13:00:00+00:00
[2025-07-24T17:44:55.878+0000] {standard_task_runner.py:60} INFO - Started process 5050 to run task
[2025-07-24T17:44:55.902+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'tune_stream_dag', 'Load_user_dim_table', 'scheduled__2023-01-07T13:00:00+00:00', '--job-id', '3160', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmps3dobjwl']
[2025-07-24T17:44:55.935+0000] {standard_task_runner.py:88} INFO - Job 3160: Subtask Load_user_dim_table
[2025-07-24T17:44:56.297+0000] {task_command.py:423} INFO - Running <TaskInstance: tune_stream_dag.Load_user_dim_table scheduled__2023-01-07T13:00:00+00:00 [running]> on host 5def55d2a1b5
[2025-07-24T17:44:56.663+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='genz-legend' AIRFLOW_CTX_DAG_ID='tune_stream_dag' AIRFLOW_CTX_TASK_ID='Load_user_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2023-01-07T13:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-07T13:00:00+00:00'
[2025-07-24T17:44:56.794+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-24T17:44:56.841+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=tune_stream_dag, task_id=Load_user_dim_table, execution_date=20230107T130000, start_date=20250724T174455, end_date=20250724T174456
[2025-07-24T17:44:57.024+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-24T17:44:57.109+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
